tests/Aspire.Hosting.MongoDB.Tests/MongoDBPublicApiTests.cs

```diff
@@ -56,7 +56,7 @@ public void AddMongoDBWithParametersShouldThrowWhenBuilderIsNull()
     [InlineData(false)]
     public void AddMongoDBWithParametersShouldThrowWhenNameIsNullOrEmpty(bool isNull)
     {
-        var builder = TestDistributedApplicationBuilder.Create();
+        var builder = TestDistributedApplicationBuilder.CreateWithTestContainerRegistry(testOutputHelper);
```

Comment:
Why all these changes to call CreateWithTestContainerRegistry when these tests don't touch the docker registry?

Reply:
@radical made this change, I think it was just a blanket pattern change.

Reply:
I have reverted the changes. But kept the ones for Qdrant, RabbitMQ, and Redis. Some of the cases were inconsistently using the test acr.